console/src/main/java/org/web3j/console/ProjectCreator.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright 2019 Web3 Labs LTD.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package org.web3j.console;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+
+public class ProjectCreator extends ProjectManager {
+
+    private String packageName;
+    private String projectName;
+    private String pathToDirectory;
+
+    private ProjectCreator(String pathToDirectory, String packageName, String projectName) {
+        this.pathToDirectory = pathToDirectory;
+        this.packageName = packageName;
+        this.projectName = projectName;
+    }
+
+    public ProjectCreator() {
+    }
+
+    public static void main(String[] args) {
+        new ProjectCreator(".", args[0], args[1]).run();
+    }
+
+    private void run() {
+
+        try {
+
+            String[] projectMainFolder = createFolderStructure(pathToDirectory, projectName);
+            String packagePath = createPackageStructure(projectMainFolder[0], packageName);
+            generateJavaClass(packagePath, projectName, packageName);
+            generateGradleBuildFile(pathToDirectory + File.separator + projectName, packageName);
+            generateGradleSettingsFile(pathToDirectory + File.separator + projectName, projectName);
+            generateSolidityContract(projectMainFolder[2]);
+            generateGradlewFiles(pathToDirectory + File.separator + projectName);
+            generateGradleWrapperPropertiesFile(projectMainFolder[3]);
+            copyWrapperJarFromResources(projectMainFolder[3]);
+            buildGradleProject(getOS(),pathToDirectory,projectName);
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
or handle appropriately - should be similar to other console arguments implementations